index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Vishal's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet" />
+
+</head>
+
+<body>
+
+  
+
+
+    <div class="wrapper">
+        <header class="nav-logo">
+
+            <a href="#" class="logo">My Logo</a>
+            <nav>
+                <img class="menu" src="./image/Union.svg" alt="Menu">
+            </nav>
+        </header>
+        <section class="content">
+            <div class="introduction">
+                <h1 class="intro">I’m Vishal <br> Glad to see <br> you! </h1>
+                <p class="description">I’m a software developer! I can help you build a product , feature or website Look
+                    through some of my work and experience! If you like what you see and have a project you need coded,
+                    don’t hestiate tocontact me.</p>
+            </div>
+    
+            <div class="social-media">
+                <a href="#" class="contact">LET'S CONNECT </a>
+                <ul class="social-link">
+                    <li class="tweet"><a href="#">
+                            <img src="image/twitter.svg" alt="My twitter">
+                        </a>
+                    </li>
+                    <li><a href="#">
+                            <img src="image/Linkedin.svg" alt="linkdin">
+                        </a>
```

Comment:
Nice work @Vishal30M your project is almost perfectly done, the only issue there is the alignment of contents in the headline section. I suggest that you make the alignment of the highlighted area match with the figma design to improve the look and feel of the website.


YOUR VERSION


![vishal](https://user-images.githubusercontent.com/84930355/212220208-073e9e5d-8fa1-403f-9bee-26990903baec.PNG)


FIGMA VERSION

![pos](https://user-images.githubusercontent.com/84930355/212220469-fcca6f55-81bb-428a-bb96-c79ca49efeba.PNG)